lib/rubygems/commands/advisory_command.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+require 'rubygems/command'
+require 'rubygems/package'
+require 'rubygems/local_remote_options'
+require 'rubygems/version_option'
+require 'rubygems/gemcutter_utilities'
+
+class Gem::Commands::AdvisoryCommand < Gem::Command
+  include Gem::VersionOption
+  include Gem::GemcutterUtilities
+
+  def initialize
+    super 'advisory', 'add advisories to gem versions'
+
+    add_version_option
+    add_platform_option
+  end
+
+  def arguments # :nodoc:
+    "GEM  name of the gem to be marked vulnerable"
+  end
+
+  def description # :nodoc:
+    <<-EOF
+The advisory command allows you to add advisories to a gem version.
+
+An advisory can be any vulnerability in the gem that makes the gem
+either unusable or risky to use. The risk can or cannot be a security issue.
+
+    EOF
+  end
+
+  def usage # :nodoc:
+    "#{program_name} advisory GEM -v VERSION [-p PLATFORM]"
```

Comment:
Remove the word advisory because it prints out "advisory" twice.  See your [screenshot](https://lifeinoss.files.wordpress.com/2018/08/screen-shot-2018-08-14-at-8-58-50-am.png).